Dockerfile

```diff
@@ -0,0 +1,106 @@
+# Build:
+#   docker build -t discourse/ruby:3.3-bookworm-slim --build-arg DEBIAN_RELEASE=bookworm --build-arg RUBY_VERSION=3.3.3 .
+#
+# You are not expected to build and push this Docker image by hand. This is done by the `build` Github Actions workflow.
+ARG DEBIAN_RELEASE=
+ARG RUBY_VERSION=
+
+FROM debian:${DEBIAN_RELEASE}-slim
+
+ARG RUBY_VERSION
+
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US.UTF-8
+ENV LC_ALL en_US.UTF-8
+ENV RUBY_VERSION=${RUBY_VERSION}
+
+# Installs system dependencies required to run Ruby
+RUN set -eux; \
+  DEBIAN_FRONTEND=noninteractive apt-get update; \
```

Comment:
I wonder if `DEBIAN_FRONTEND=noninteractive` could be another `ENV` listed above (together with lines 12-15). Then we can remove all the other mentions of it.

Reply:
Actually I think I can just remove `DEBIAN_FRONTEND=noninteractive`, I don't think it is required.